components/learn/boxes/Boxes.js

```diff
@@ -137,8 +137,7 @@ export default function Boxes(props) {
                           <a href={`${prefix}/learn/graphql-tool`} className={styles.titleLink}>
                             GraphQL tool
                           </a>
-                        </p>
-                        <p className={styles.description}>Generate GraphQL client skeletons in Ballerina.</p>
+                        <p className={styles.description}>Generate GraphQL service skeletons for a given GraphQL schema and generate schema for a given Ballerina GraphQL service.</p>
```

Comment:
```suggestion
                        <p className={styles.description}>Generate the Ballerina GraphQL service skeleton for a given GraphQL schema and vice versa.</p>
```

Reply:
IMO it's better to describe both operations here. BTW we are generating the schema from the service, not from the service skeletons. In that case, can we use `vice versa`?

Reply:
+1